pb/get_configuration.go

```diff
@@ -58,17 +58,38 @@ func (c *RemoteProvisioning) Clone() *RemoteProvisioning {
 	}
 }
 
+func (c *BuildInfo) Clone() *BuildInfo {
+	if c == nil {
+		return nil
+	}
+	return &BuildInfo{
+		Version:    c.GetVersion(),
+		BuildDate:  c.GetBuildDate(),
+		CommitHash: c.GetCommitHash(),
+		CommitDate: c.GetCommitDate(),
+		ReleaseUrl: c.GetReleaseUrl(),
+	}
+}
+
+func NewGetConfigurationResponse(info *BuildInfo) *GetConfigurationResponse {
+	v := &GetConfigurationResponse{
+		BuildInfo: info,
+	}
+	return v.Clone()
```

Comment:
Why clone a locally created object? Because of the `info` pointer?

Reply:
Just to fill deprecated values. Fixed.